NitroxClient/GameLogic/Crafting.cs

```diff
@@ -9,27 +12,453 @@ namespace NitroxClient.GameLogic
 {
     public class Crafting
     {
+        internal static Crafting Instance; 
```

Comment:
We should not use a singleton here, instead opting to use our dependency injection framework AutoFac.  Please review how the NItroxServiceLocater is used in other places.

Reply:
Okay, i will change this.